pa_bin/performance-analyzer-agent

```diff
@@ -15,14 +15,23 @@ else
 fi
 
 if [ -z "$2" ]; then
-    if [ -z "$JAVA_HOME" ]; then
-        echo "JAVA_HOME needs to be set or passed in as the second parameter."
+  # Check if OPENSEARCH_JAVA_HOME is present
+  if [ -z "$OPENSEARCH_JAVA_HOME" ]; then
+      # If OPENSEARCH_JAVA_HOME is not present, check JAVA_HOME next
+      if [ -z "$JAVA_HOME" ]; then
+        echo "OPENSEARCH_JAVA_HOME / JAVA_HOME needs to be set or passed in as the second parameter."
         exit 1
-    fi
+      fi
+  else
+    JAVA_HOME=$OPENSEARCH_JAVA_HOME
+  fi
 else
     JAVA_HOME=$2
 fi
 
+echo "Using JAVA_HOME: $JAVA_HOME"
```

Comment:
See please https://github.com/opensearch-project/performance-analyzer/pull/133#discussion_r801062295 for the context for this change